charters/DataBrowser/charter.md

```diff
@@ -0,0 +1,67 @@
+# Data Browser
+
+
+## Description
+The Data Browser (available via the Data Portal 'Explore' section) is an application that provides a faceted search capability allowing users to search and download subsets of cellular resolution data hosted by the Data Coordination Platform (DCP).
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyzes, and make available the data in the Human Cell Atlas project.
+
+*Data Portal* The landing site for the Human Cell Atlas (HCA) DCP.
+
+*Analysis Applications* Third party portals listed in the 'Analyze' section of the Data Portal UI.
+
+## Objectives
```

Comment:
I'll ref [my comment](https://github.com/HumanCellAtlas/dcp-community/pull/10#discussion_r223229320) on the portal charter here.

I want to see an indication somewhere of who is responsible for providing users of data from the DCP a sensible view over that data. Over time, metadata and data will change - in terms of schema, data file formats, QC, analysis results generated from new versions of pipelines. Where in the DCP is this wrangled into a single up-to-date view to provide some guarantees to users over the quality and contemporaneity of the data they get from us?

Reply:
A larger DCP-wide discussion outside the remit of just this charter.